pkg/analyzer/parser.go

```diff
@@ -297,10 +308,15 @@ func (p *NSXConfigParser) getEndpointsFromGroupsPaths(groupsPaths []string, excl
 	vms := []endpoints.EP{}
 	groups := []*collector.Group{}
 	if exclude {
-		groupsPaths = slices.DeleteFunc(slices.Clone(p.allGroupsPaths), func(p string) bool { return slices.Contains(groupsPaths, p) })
+		// TODO: what should we do with IP Addresses in groupsPaths and exclude
```

Comment:
currently add a warning in such case that this is not supported / ignored

Reply:
done